workspace/libs/shared/ui/src/lib/modules/dashboard/components/dashboard-v2/dashboard-v2.component.scss

```diff
@@ -138,7 +138,7 @@ h1 {
 }
 
 .custom-mat-sidenav-content {
-  overflow: visible;
+  overflow: hidden;
```

Comment:
This applies to all layouts, does it make more sense to make this change under a more specific selector since this is a mobile (viewport-xsmall) issue?